Bottom: 9cab99596df6281dda23cd8acb2add1dd0344956
Top:    2be4ba096dd79076b5a6723f7a7b3f138d85311e
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-07-24 21:39:43 +0800

Refresh of tun-use-build_skb-for-small

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index afe311f..5652bf3 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1301,6 +1301,7 @@ static ssize_t tun_get_user(struct tun_struct *tun, struct tun_file *tfile,
 			zerocopy = true;
 	}
 
+	/* FIXME: check DONTWAIT and INT_MAX */
 	if (SKB_DATA_ALIGN(len + TUN_RX_PAD) +
 	    SKB_DATA_ALIGN(sizeof(struct skb_shared_info)) < PAGE_SIZE) {
 		skb = tun_build_skb(tfile, from, len);
